src/stamp/preprocessing/extractor/h_optimus_1.py

```diff
@@ -0,0 +1,30 @@
+try:
+    import timm 
+    from torchvision import transforms
+except ModuleNotFoundError as e:
+    raise ModuleNotFoundError(
+        "h_optimus_1 dependencies not installed."
+        " Please reinstall stamp using `pip install 'stamp[h_optimus_1]'`"
+    ) from e
```

Comment:
Please add these dependencies as a dependency group in the toml file: https://github.com/EzicStar/STAMP/blob/0b4760cdecd71e04cad5667d450ccbab70bbbec2/pyproject.toml#L47-L72